### PR TITLE
Release/v1.6.0 rc1

### PR DESCRIPTION
**Pull request by arnabmitra**
## Description

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by arnabmitra** [comment]
once 1.6 is forked from 1.5, move base branch there.

**Comment by arnabmitra** [comment]
testing notes 
original state:
```

provenance on  tags/v1.5.0 [$!] at ☸️ gke_figure-development_us-east1-b_tf-test took 5s
➜ provenanced -t query marker get usdf.c
marker:
  '@type': /provenance.marker.v1.MarkerAccount
  access_control: []
  allow_governance_control: true
  base_account:
    account_number: "11"
    address: tp1c9pv3n0pmjezg0wkgwqze00957ve39hwsjy84h
    pub_key: null
    sequence: "0"
  denom: usdf.c
  manager: tp125rltkc8kmkm4hcdnl7uda9deau9aldryh089c
  marker_type: MARKER_TYPE_COIN
  status: MARKER_STATUS_PROPOSED
  supply: "0"
  supply_fixed: false
```
after `"usdf.c-hotfix"`
```provenance on  release/v1.6.0-RC1 [$!] at ☸️ gke_figure-development_us-east1-b_tf-test
➜ provenanced -t query marker get usdf.c
marker:
  '@type': /provenance.marker.v1.MarkerAccount
  access_control: []
  allow_governance_control: true
  base_account:
    account_number: "11"
    address: tp1c9pv3n0pmjezg0wkgwqze00957ve39hwsjy84h
    pub_key: null
    sequence: "0"
  denom: usdf.c
  manager: tp125rltkc8kmkm4hcdnl7uda9deau9aldryh089c
  marker_type: MARKER_TYPE_RESTRICTED
  status: MARKER_STATUS_PROPOSED
  supply: "0"
  supply_fixed: false
```
@iramiller we should probably gov propose `usdc.f`on `pio-test` so that it runs there before prod

File: x/marker/keeper/keeper_test.go [hunk 1]
```diff
@@ -128,6 +128,88 @@ func TestAccountKeeperReader(t *testing.T) {
...
+
+	m, err = app.MarkerKeeper.GetMarker(ctx, addr)
+	require.NoError(t, err)
+	require.NotNil(t, m)
+	require.EqualValues(t, m.GetDenom(), "usdf.c")
+	require.EqualValues(t, m.GetAddress(), addr)
+	require.EqualValues(t, m.GetMarkerType(), types.MarkerType_Coin)
+
+	count := 0
+	app.MarkerKeeper.IterateMarkers(ctx, func(record types.MarkerAccountI) bool {
```
Comment (iramiller) [inline x/marker/keeper/keeper_test.go:160]: we don't really need this check as denom is enforced to be the address of the marker and thus must be unique on chain as address is the primary key/identifier for accounts.

Comment (arnabmitra) [reply x/marker/keeper/keeper_test.go:160]: this was just an additional check that the earlier tests were doing ...i can remove it 🤷

**Comment by arnabmitra** [comment]
moving to another PR

**Pull request closed without merging**